hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/snapshot/OzoneManagerSnapshotProvider.java

```diff
@@ -176,6 +177,14 @@ public DBCheckpoint getOzoneManagerDBSnapshot(String leaderOMNodeID)
 
         long snapshotIndex = Long.parseLong(header.getValue());
 
+        header = response.getFirstHeader(OM_RATIS_SNAPSHOT_TERM_INDEX);
+        if (header == null) {
```

Comment:
This can be null in case where setBothHeaders is false in OMDBCheckpointServlet. Do we always want to set both the headers?

Reply:
Yes, when OM follower downloads checkpoint, we set both the headers.